### PR TITLE
Use libutil provided functions in zdev

### PR DESCRIPTION
**Pull request by r4f4**

### CODE REVIEW

**Comment by oberpar** [comment]
Looks good. We're losing some debug output but those calls can be replaced by using strace for debugging.

`Acked-by: Peter Oberparleiter <oberpar@linux.vnet.ibm.com>`

File: libutil/util_path.c [hunk 1]
```diff
@@ -194,3 +194,15 @@ bool util_path_is_dir(const char *fmt, ...)
 	free(path);
 	return rc;
 }
+
+bool util_path_exists(const char *fmt, ...)
```
Comment (michael-holzheu) [inline libutil/util_path.c]: Looks good to me: Acked-by: Michael Holzheu <holzheu@linux.vnet.ibm.com>

**Comment by michael-holzheu** [comment]
```
s390-tools $ make
  CC      zdev/src/devtype.o
devnode.c: In function ‘add_block_cb’:
devnode.c:233:2: warning: implicit declaration of function ‘util_path_is_dir’ [-Wimplicit-function-declaration]
  if (util_path_is_dir(path))
  CC      zdev/src/lcs.o
ctc_auto.c: In function ‘read_sorted_ctc_devinfos’:
ctc_auto.c:128:2: warning: implicit declaration of function ‘util_path_is_dir’ [-Wimplicit-function-declaration]
  if (util_path_is_dir(path))
```
@r4f4 Come on ... at least a compile test should be mandatory, no? Or am I missing something?

**Comment by r4f4** [comment]
@michael-holzheu my bad, when doing the rebase I skipped one of my commits.

**Comment by michael-holzheu** [comment]
> @michael-holzheu my bad, when doing the rebase I skipped one of my commits.

Looks better now. Applied, thanks!

**Pull request closed without merging**